Resources/parameters.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
You can manage the indentation of json braces in VS Code, these doc looks like it's set to 2.  Personally I find 4 easier to read